### ISSUE TITLE
Hero image is strangely off-balance with large viewports (>=`1440px`)

### ISSUE THREAD

**Issue by tomayac** [GoogleChrome/developer.chrome.com#2747]
Note how the hero image is strangely position too far on the right. This happens when the screen hits the `1440px` media query.

<img width="1164" alt="Screen Shot 2022-05-03 at 11 53 20" src="https://user-images.githubusercontent.com/145676/166434611-5281ff27-8410-41e8-8ccf-b39232d4e382.png">
<img width="1117" alt="Screen Shot 2022-05-03 at 11 53 30" src="https://user-images.githubusercontent.com/145676/166434635-eef72440-0fa4-441e-8429-f60c643c6453.png">

**Comment by jeffposnick** [comment]
This seems like the same issue as https://github.com/GoogleChrome/developer.chrome.com/issues/2747

We can use that for follow-up, assuming you agree that it's the same issue.

**Comment by jeffposnick** [comment]
I took a look at the underlying CSS, and there is heavy use of flexbox-based centering for those elements.

The hero image looks properly centered, but the centering of the article body is off when the TOC element is displayed to the left of the article. This is because the TOC element has an explicit width of 20rem, which contributes to extra whitespace being taken into account when centering the article.

Removing the explicit `width: 20rem` on the TOC container ends up leading to good-looking results:
![Screen Shot 2022-06-23 at 2 29 24 PM](https://user-images.githubusercontent.com/1749548/175369450-5807dc28-55a3-4fe3-b55d-9864f00328d5.png)

If there are very long items in the TOC, then the article's margins do end up being shifted further to the edges than they otherwise would be, but at least it "looks" centered in that case (i.e. it's actual text, rather than whitespace, that's contributing to that shift):
![Screen Shot 2022-06-23 at 2 29 08 PM](https://user-images.githubusercontent.com/1749548/175369692-35728590-302e-4fd8-ae38-5686c60d3908.png)

I'll file a PR to remove the `width: 20rem` on the TOC container. I don't know the history behind why it was added, so I'll ask some folks who might know to perform the PR review.
